pallets/cash/src/reason.rs

```diff
@@ -50,8 +50,8 @@ pub enum Reason {
     RatesError(RatesError),
     RepayTooMuch,
     SelfTransfer,
-    SetYieldNextError(SetYieldNextError),
     SerdeError,
+    SetYieldNextError(SetYieldNextError),
```

Comment:
We should really stop alphabetizing this, since it makes types.json hard to stay consistent.